test/report/report.test.ts

```diff
@@ -165,7 +240,23 @@ describe("report", () => {
 
                 return mockedReportModule()
                     .exportResult(projectId, executionResult, exportConfig, { pollStep: 1 })
-                    .then((result: IExportResponse) => expect(result.uri).toEqual(createdReport));
+                    .then((result: IExportResponse) => {
+                        expect(result.uri).toEqual(createdReport);
+
+                        const [, settings] = fetchMock.lastCall(
```

Comment:
this looks weird, does it work? `[_, settings] = ...` is not this better?

Reply:
It is described here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Ignoring_some_returned_values
I tried with an underscore but it does not work and requires "TS warning disable" for unused variable. I cut and pasted this part from one of the other tests in the file 😅

Reply:
I thought so